### PR TITLE
Fixup mountpoints z3c

### PR DESCRIPTION
**Pull request by rss351**
Need to include additional partitions for sony z3c/aries in the fixup-mountpoints script.

### CODE REVIEW

**Comment by mlehtima** [comment]
Please combine the commits to a single commit.

**Comment by rss351** [comment]
I merged the commits for fixup-mountpoints into eec57ea13e316ac2150599cd19498c95a17b1416.  Let me know if this didn't work. Thanks!

File: fixup-mountpoints [hunk 1]
```diff
@@ -115,6 +115,36 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/LTALabel mmcblk1 ' \
             "$@"
         ;;
+    "z3c" | "d5803" | "aries")
```
Comment (sledges) [inline fixup-mountpoints]: A note for future: Xiaomi Mi 2/2S has codename "aries", so we'll probably need some CM/AOSP detection

**Comment by sledges** [comment]
Pls rename commit message to ```[z3c][aries] Add fixup-mountpoints for CM/AOSP Xperia Z3 Compact```

File: fixup-mountpoints (hunk 1 above)
Comment (maikelwever) [inline fixup-mountpoints]: We can omit the 'aries' and 'd5803' for now, the CM12 based build has codename 'z3c' so that will work.

**Comment by rss351** [comment]
Renamed the commit message.  Thanks!

**Comment by sledges** [comment]
LGTM :+1: great team effort!

**Comment by sledges** [comment]
@rss351 would you like to remove aries and d5803 as per what @maikelwever said?

**Comment by rss351** [comment]
Sorry for the delay.  It is updated.  Thanks!

**Comment by sledges** [comment]
thanks! :+1: merging

**Pull request merged**